index/miner/onchain.go

```diff
@@ -31,6 +31,9 @@ func (mi *MinerIndex) updateOnChainIndex() error {
 	if err != nil {
 		return err
 	}
+	if heaviest.Height()-hOffset <= 0 {
+		return nil
+	}
```

Comment:
Cover an edge case that only happens when the indexes are running when a super-fresh lotus node is syncing from genesis.